tox.ini

```diff
@@ -0,0 +1,63 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
+
+toxworkdir={toxinidir}/.tox
+
+[testenv]
+commands_pre = 
+    pip install -r {toxinidir}/requirements/dev.txt
+    pip install -r {toxinidir}/requirements/common.txt
+    pip install -r {toxinidir}/requirements/docs.txt
+commands =
+    pip check
+    {toxinidir}/lints/queuelint.py
+    flake8 --show-source
+    isort --check-only --diff --quiet
+    mkdocs build
+    pre-commit run --all-files shellcheck
+    pre-commit run --all-files markdownlint
+
+[testenv:virtualenv]
```

Comment:
Merge the `virtualenv` environment into the default one.

Reply:
I wanted to keep it different so that we have seperate of runchecks.sh and rest everything different. But I can merge them!